app/src/main/scala/org/alephium/explorer/persistence/queries/ExplainResult.scala

```diff
@@ -25,4 +25,5 @@ object ExplainResult {
 final case class ExplainResult(queryName: String,
                                queryInput: String,
                                explain: Vector[String],
+                               message: Iterable[String],
```

Comment:
a bit counter-intuitive to have an `Iterable` while using `None` as value. I would prefer using an `Option` here, also for the JSON serialization, as the `Iterable` will be a `[]` while `Option` is `null`. We want the second.

Reply:
Switched `Option[String]`.

The reason to use a `Iterable[String]` was so that the output json is more readable. Just like how `explain` being a `Vector[String]` renders the text in JSON just like how Postgres renders it in console. 

## Currently `Iterable[String]` gets this which is more readable 
![image](https://user-images.githubusercontent.com/1773953/182509669-9e93aae7-91c9-4f66-972e-e5babb9007d8.png)

### Using `Option[String]` get this
![image](https://user-images.githubusercontent.com/1773953/182509227-de383a63-b98d-4e02-99fd-8aca3eba85a7.png)

### Using `Option[String]` without newline get this
![image](https://user-images.githubusercontent.com/1773953/182509572-7453d092-cfb9-4a9b-8807-1e2abe60ad66.png)

Reply:
ooooooo my bad, this is indeed way more readable to use the `Iterable` + it makes sense as each item is another message. Then maybe we could go back to `Iterable` but rename to `messages`? HDYT? 
sorry for this back and forth

Reply:
No worries at all :) Will do. Yep `messages` is clearer will make this change.